src/table/internal.tsx

```diff
@@ -269,7 +272,7 @@ const InternalTable = React.forwardRef(
           }
           __stickyHeader={stickyHeader}
           __stickyOffset={
-            isMobile ? (stickyHeaderVerticalOffset ?? 0) - mobileStickyOffset : stickyHeaderVerticalOffset
+            isMobile ? (stickyHeaderVerticalOffset ?? 0) - Math.abs(mobileStickyOffset) : stickyHeaderVerticalOffset
```

Comment:
Why do we use `Math.abs`? If we want to avoid negative values, could we use `Math.max(mobileStickyOffset, 0)` instead?